2021-06-15 03:31:46,634 INFO    MainThread:1301489 [internal.py:wandb_internal():91] W&B internal server running at pid: 1301489, started at: 2021-06-15 03:31:46.634318
2021-06-15 03:31:46,644 DEBUG   HandlerThread:1301489 [handler.py:handle_request():124] handle_request: check_version
2021-06-15 03:31:46,646 INFO    WriterThread:1301489 [datastore.py:open_for_write():80] open: /home/chengzilong/DDQN/wandb/run-20210615_033145-1upywtte/run-1upywtte.wandb
2021-06-15 03:31:46,651 DEBUG   SenderThread:1301489 [sender.py:send():179] send: header
2021-06-15 03:31:46,652 DEBUG   SenderThread:1301489 [sender.py:send_request():193] send_request: check_version
2021-06-15 03:31:46,722 DEBUG   SenderThread:1301489 [sender.py:send():179] send: run
2021-06-15 03:31:46,744 DEBUG   SenderThread:1301489 [git.py:repo():33] git repository is invalid
2021-06-15 03:31:47,046 INFO    SenderThread:1301489 [dir_watcher.py:__init__():168] watching files in: /home/chengzilong/DDQN/wandb/run-20210615_033145-1upywtte/files
2021-06-15 03:31:47,047 INFO    SenderThread:1301489 [sender.py:_start_run_threads():707] run started: 1upywtte with start time 1623699105
2021-06-15 03:31:47,047 DEBUG   SenderThread:1301489 [sender.py:send():179] send: summary
2021-06-15 03:31:47,049 INFO    SenderThread:1301489 [sender.py:_save_file():829] saving file wandb-summary.json with policy end
2021-06-15 03:31:47,049 DEBUG   HandlerThread:1301489 [handler.py:handle_request():124] handle_request: run_start
2021-06-15 03:31:48,064 INFO    Thread-14 :1301489 [dir_watcher.py:_on_file_created():216] file/dir created: /home/chengzilong/DDQN/wandb/run-20210615_033145-1upywtte/files/wandb-summary.json
2021-06-15 03:31:49,524 DEBUG   HandlerThread:1301489 [meta.py:__init__():39] meta init
2021-06-15 03:31:49,524 DEBUG   HandlerThread:1301489 [meta.py:__init__():53] meta init done
2021-06-15 03:31:49,524 DEBUG   HandlerThread:1301489 [meta.py:probe():210] probe
2021-06-15 03:31:49,530 DEBUG   HandlerThread:1301489 [git.py:repo():33] git repository is invalid
2021-06-15 03:31:49,531 DEBUG   HandlerThread:1301489 [meta.py:_save_pip():57] save pip
2021-06-15 03:31:49,533 DEBUG   HandlerThread:1301489 [meta.py:_save_pip():71] save pip done
2021-06-15 03:31:49,533 DEBUG   HandlerThread:1301489 [meta.py:_save_conda():78] save conda
2021-06-15 03:31:50,073 INFO    Thread-14 :1301489 [dir_watcher.py:_on_file_created():216] file/dir created: /home/chengzilong/DDQN/wandb/run-20210615_033145-1upywtte/files/conda-environment.yaml
2021-06-15 03:31:50,074 INFO    Thread-14 :1301489 [dir_watcher.py:_on_file_created():216] file/dir created: /home/chengzilong/DDQN/wandb/run-20210615_033145-1upywtte/files/requirements.txt
2021-06-15 03:31:50,802 WARNING MainThread:1301489 [internal.py:wandb_internal():147] Internal process interrupt: 1
2021-06-15 03:31:50,865 DEBUG   HandlerThread:1301489 [meta.py:_save_conda():86] save conda done
2021-06-15 03:31:50,865 DEBUG   HandlerThread:1301489 [meta.py:probe():252] probe done
2021-06-15 03:31:50,877 DEBUG   SenderThread:1301489 [sender.py:send():179] send: files
2021-06-15 03:31:50,878 INFO    SenderThread:1301489 [sender.py:_save_file():829] saving file wandb-metadata.json with policy now
2021-06-15 03:31:50,974 WARNING MainThread:1301489 [internal.py:wandb_internal():147] Internal process interrupt: 2
2021-06-15 03:31:50,974 ERROR   MainThread:1301489 [internal.py:wandb_internal():150] Internal process interrupted.
2021-06-15 03:31:51,073 INFO    Thread-14 :1301489 [dir_watcher.py:_on_file_created():216] file/dir created: /home/chengzilong/DDQN/wandb/run-20210615_033145-1upywtte/files/wandb-metadata.json
2021-06-15 03:31:51,312 INFO    MainThread:1301489 [internal.py:handle_exit():78] Internal process exited
